src/server/dcp_server/utils.py

```diff
@@ -72,9 +76,12 @@ def crop_centered_padded_patch(x: np.ndarray,
         x[m] = 0
```

Comment:
I can't add it above, but please include mask and noise_intensity in function args documentation